.github/workflows/runtests.yml

```diff
@@ -114,53 +114,39 @@ jobs:
       - run: bin/doctest --force-colors
       - run: examples/all.py -q
 
-  # -------------------- Test split 1/2 latest Python -------------- #
+  # ------------------------- Test latest Python ------------------- #
 
-  tests1-latest:
-    needs: code-quality
-
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-      - run: python -m pip install --upgrade pip
-      - run: pip install mpmath
-      - run: bin/test --force-colors --split=1/2
-
-  # -------------------- Test split 2/2 latest Python -------------- #
-
-  tests2-latest:
+  tests-latest:
     needs: code-quality
 
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [1, 2, 3, 4]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
           python-version: '3.10'
       - run: python -m pip install --upgrade pip
-      - run: pip install mpmath
-      - run: bin/test --force-colors --split=2/2
+      - run: pip install mpmath pytest pytest-split pytest-xdist
+      - run: bin/test --split ${{ matrix.group }}/4
 
   # -------------------- Test Pyodide on node ---------------------- #
 
   test-pyodide:
     needs: code-quality
 
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        split: ['1/2', '2/2']
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 16.x
       - run: wget -qO- https://github.com/pyodide/pyodide/releases/download/0.20.1a1/pyodide-build-0.20.1a1.tar.bz2 | tar xjf -
       - run: python3 setup.py bdist_wheel
-      - run: node bin/test_pyodide.mjs --split=${{ matrix.split }} 2>/dev/null  # ignore node exception
+      - run: node bin/test_pyodide.mjs 2>/dev/null  # ignore node exception
```

Comment:
I'm not sure why stderr is redirected here but maybe at least temporarily it would help to remove the redirect.

Reply:
Because when python throws an exception node will also throw, whose log is too long and doesn't provide useful information. When we are sure the exception comes from python instead of node, we can use the redirect.